src/rime/gear/script_translator.cc

```diff
@@ -431,7 +431,7 @@ bool ScriptTranslation::Next() {
   } while (enable_correction_ &&
            syllabifier_->IsCandidateCorrection(*candidate_) &&
            // limit the number of correction candidates
-           ++correction_count_ > max_corrections_);
```

Comment:
This while-loop is used to drop any correction candidates after the top N (`max_corrections_`). 

If I didn't take it wrong, `correction_count_ > max_corrections_` means exactly "correction candidates after the top N" so the loop goes on.

I recently updated this part. I might have broken it. Please blame and review my last change.
One thing that appear weird is the `is_correction` never set true.